### PR TITLE
(+) python script to extract one particle history from an HDF5 file

### PR DESCRIPTION
**Pull request by korobkin**
Usage example:
./extract_particle.py sedov_evolution.h5part -i 100 > particle_00100.dat

### CODE REVIEW

File: tools/extract_particle.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
```
Comment (junghans) [inline tools/extract_particle.py]: Does it work with python3?

Comment (korobkin) [reply tools/extract_particle.py]: No.

Comment (korobkin) [reply tools/extract_particle.py]: We need to make sure all our scripts are compatible with Python3.

Comment (junghans) [reply tools/extract_particle.py]: https://pythonclock.org/, you have 1 year 1 month ;-)

**Pull request merged**